ci-operator/jobs/openshift/windows-machine-config-operator/openshift-windows-machine-config-operator-master-presubmits.yaml

```diff
@@ -107,30 +125,30 @@ presubmits:
       - name: result-aggregator
         secret:
           secretName: result-aggregator
-    trigger: (?m)^/test( | .* )e2e-operator,?($|\s.*)
+    trigger: (?m)^/test( | .* )aws-e2e-operator-with-custom-vxlan-port,?($|\s.*)
   - agent: kubernetes
-    always_run: true
+    always_run: false
```

Comment:
Why was this made optional?

Reply:
It will be made required once https://github.com/openshift/windows-machine-config-operator/pull/106 is ready to merge. It is done this way to help @LorbusChris move forward.